app/views/shared/_contact_panels.html.erb

```diff
@@ -28,9 +28,17 @@
         </div>
 
         <ul class="t-chat-opening-times contact-panel__list unstyled-list">
-          <% chat_opening_hours.open_periods.each do |period| %>
-            <li class="contact-panel__additional-info"><%= period.html_safe %></li>
+
+          <% unless migration_active? %>
```

Comment:
I would change the method name. Migration is too generic, could have different meanings.